org.opent2t.sample.binaryswitch.superpopular/com.wink.binaryswitch/js/thingTranslator.js

```diff
@@ -88,7 +88,7 @@ class Translator {
 
         // Initialize Wink Helper
         winkHelper = new WinkHelper(device.props.access_token);
-        console.log('Javascript and Wink Helper initialized : ');

```

Comment:
Likely that will be a good investment... only we have to ensure that the open spirit of this project is respected. We should give the developer control over how they want to log... and not pick any particular telemetry / analytics engine. Please participate here with your ideas: https://github.com/openT2T/translators/issues/60. For this PR, this is #WontFix but this will be a great area to get your help in! :)